resources/core/default-browser.json

```diff
@@ -53,7 +53,8 @@
         "RenderingEngine_Name": "unknown",
         "RenderingEngine_Version": "unknown",
         "RenderingEngine_Description": "unknown",
-        "RenderingEngine_Maker": "unknown"
+        "RenderingEngine_Maker": "unknown",
+        "PatternId": "resources/core/default-browser.json::u0"
```

Comment:
I needed to add a hard-coded pattern id to these two files in order for the pattern ids to appear in the other patterns (the code checks that default properties specifically has the key). The changes I've made to the BuildHelper class will remove these if PatternId collection isn't enabled.